clang/www/cxx_status.html

```diff
@@ -356,14 +356,7 @@ <h2 id="cxx23">C++23 implementation status</h2>
     <tr>
       <td>Relaxing some constexpr restrictions</td>
       <td><a href="https://wg21.link/P2448R2">P2448R2</a></td>
-      <td class="partial" align="center">
-        <details><summary>Clang 17 (Partial)</summary>
-	  We do not support outside of defaulted special memeber functions the change that constexpr functions no
-          longer have to be constexpr compatible but rather support a less restricted requirements for constexpr
-          functions. Which include allowing non-literal types as return values and parameters, allow calling of
-          non-constexpr functions and constructors.
-        </details></td>
-      </td>
+      <td class="unreleased" align="center">Clang 18</td>
```

Comment:
I suppose, I should put clang 19 here, right?

Reply:
@AaronBallman WDYT?

Reply:
Discussed on the call, this is not that important to cherry pick to 18, so I'll change this to 19.